sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+  being thrown instead of empty vector being returned. intel/llvm#13968
+- Fixed `esimd::atan` implementation under `-ffast-math` flag. intel/llvm#13186
+- Fixed an issue that component devices were not considered to be a descendent
+  from their composite devices when creating a queue. intel/llvm#13513
+- Fixed an issue that querying for
+  `ext::oneapi::experimental::info::device::composite_device` would *not*
+  throw an exception if device is not a component device. intel/llvm#13868
+- Fixed an issue that querying for composite devices may result in some devices
+  returned twice. intel/llvm#14442
+- Fixed a bug that querying for composite devices may result in an exception
+  being thrown instead of an empty vector being returned. intel/llvm#13931
+- Fixed a bug in copy-constructor of `config_2d_mem_access` ESIMD class which
+  would lead to compilation errors. intel/llvm#13632
+- Fixed an issue that use of `atomic_ref<T *>` would not be detected as a use
+  of `atomic64` aspect leading to errors due to speculative compilation. intel/llvm#14052
+- Fixed `ctanh` and `cexp` returning incorrect value in some edge cases. intel/llvm#14329
+- Fixed a bug where values passed to `-Xs` option through `build_options`
+  property were not passed down to device compiler when using
+  [`sycl_ext_oneapi_kernel_compiler`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc)
+  extension. intel/llvm#14522
+- Fixed a bug in
+  [`sycl_ext_oneapi_kernel_compiler`](https://github.com/intel/llvm/blob/ebb3b4a21b3b0e977f44434781729df7de83e436/sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_compiler.asciidoc)
+  extension implementation
+  TODO: add description here. @cperkinsintel . intel/llvm#14490
+- Fixed a bug where defining kernel as a named functor whilst using
+  `-fno-sycl-unnamed-lambda` would lead to a compilation error about unnamed
+  lambdas being unsupported. intel/lvm#14614
+- Fixed an issue on CUDA & AMDGPU backends where `multi_ptr` relational
+  operators taking `std::nullptr_t` would produce different result comparing
+  to standard C++ helpers like `std::less`. intel/llvm#13201
+- Fixed a compilation issue with `-fpreview-breaking-changes` flag when
+  `windows.h` is included caued by conflict with `min`/`max` macro. intel/llvm#14260
+- Fixed strict alias violations in `sycl::vec<sycl::half, N>::operator[]`
+  implementation that could lead to spurious errors. intel/llvm#13596
+- Fixed a bug where a barrier submitted into a command queue with host tasks
+  could ignore them, as well as a few bugs related to synchronization of host
+  tasks with barriers. intel/llvm#13094 intel/llvm#13863 intel/llvm#13094
+- Fixed a compilation issue occurring when `printf` is used on CUDA backend
+  on Windows. intel/llvm#13784
+  TODO: was it really a compilation issue?
+- Fixed an issue where compiler could emit SPIR-V instructions for reversing
```

Comment:
```suggestion
- Fixed an issue where the compiler could emit SPIR-V instructions for reversing
```

Reply:
Applied in 1fe2c554292d3a9d90f952c02ff4b8e3dfaab706